src/query-builder/update-result.ts

```diff
@@ -1,5 +1,14 @@
 export class UpdateResult {
+  /**
+   * The number of rows the update query updated.
+  */
   readonly numUpdatedRows: bigint
+
+  /**
+   * The number of rows the update query changed.
+   * This is **optional** and only provided by some drivers like `node-mysql2`.
```

Comment:
```suggestion
   * This is **optional** and only supported in dialects such as MySQL.
```

Reply:
Would you suggest the same change for [`database-connection.ts`](https://github.com/kysely-org/kysely/blob/deea8e22aae3dc4e142f055b6ec4d9a50a402bda/src%2Fdriver%2Fdatabase-connection.ts#L23-L34)?

https://github.com/kysely-org/kysely/blob/deea8e22aae3dc4e142f055b6ec4d9a50a402bda/src%2Fdriver%2Fdatabase-connection.ts#L23-L34

Reply:
Yes. changed rows also exists in `mysql`, not just `mysql2`, and its probably part of the MySQL protocol in general.

Reply:
Applied.